newscript.go

```diff
@@ -125,15 +147,15 @@ func blankBootstick() error {
 			break
 		}
 	}
-	fmt.Printf("Running dd to put the new image onto the desired location. \n")
+	fmt.Printf("Running cp to put the new image onto the desired location. \n")
 	if err := cp(imageLoc, location); err != nil {
 		return err
 	}
 	return nil
 }
 
 func cleanup() error {
-	filesToRemove := [...]string{linuxVersion, "linux-stable", "NiChrome", "vboot_reference"}
+	filesToRemove := [...]string{*linuxVersion, "linux-stable", "NiChrome", "vboot_reference"}
 	fmt.Printf("-------- Removing problematic files %v\n", filesToRemove)
 	for _, file := range filesToRemove {
 		if _, err := os.Stat(file); err != nil {
```

Comment:
so you're going to exit this loop on the first nonexistent file. Is that what you want? It means it won't remove the other files even if they exist.